sentinelhub/api/ogc.py

```diff
@@ -416,7 +416,13 @@ def _get_common_url_parameters(request):
                 if geometry.crs is CRS.WGS84:
                     geometry = geometry.reverse()
 
-                params[CustomUrlParam.GEOMETRY.value] = geometry.wkt
+                custom_params[CustomUrlParam.GEOMETRY] = geometry.wkt
+
+            for resampling in (CustomUrlParam.DOWNSAMPLING, CustomUrlParam.UPSAMPLING):
+                if resampling in custom_params:
+                    custom_params[resampling] = ResamplingType(custom_params[resampling]).value
```

Comment:
While all newer SH APIs are case-sensitive about parameter values, the old OGC is not. Therefore, it worked even with a value `"bicubic"` or even `"BiCuBiC"`. But now with this enum parsing it won't work anymore.

I'm wondering if we should add an extra check here that would convert a string to uppercase. Another option would be to even add a simple metaclass for `ResamplingType` where converting a string to uppercase would become a part of casting to enum. We already know how to do this because we do it for `CRS` enum.

Reply:
I would prefer to follow the wisdom "if you think you need a meta-class, you probably don't". I think it's best to solve this via function (or method) `parse_resampling_type` or by in-lining the parsing in the OGC and leaving it case sensitive elsewhere.